website/docs/r/amplify_domain_association.html.markdown

```diff
@@ -67,11 +67,11 @@ The `sub_domain` configuration block supports the following arguments:
 This resource exports the following attributes in addition to the arguments above:
 
 * `arn` - ARN for the domain association.
-* `certificate_verification_dns_record` - The DNS record for certificate verification.
+* `certificate_verification_dns_record` - The DNS records for certificate verification in the space-delimited format "<record> CNAME <target>"
 
 The `sub_domain` configuration block exports the following attributes:
 
-* `dns_record` - DNS record for the subdomain.
+* `dns_record` - DNS record for the subdomain in in a space prefixed and space delimited format " CNAME <target>"
```

Comment:
- Similar to the suggestion above with regards to preformatted text.
- Added dashes between space-formatted and space-prefixed.

```suggestion
* `dns_record` - DNS record for the subdomain in a space-prefixed and space-delimited format ` CNAME <target>`
```